README.md

```diff
@@ -7,17 +7,95 @@ copyright: Copyright (c) 2010-2020 K Team. All Rights Reserved.
 
 _K-based tool users_ should:
 
 1.  Consult their tool documentation for build/installation instructions.
 2.  If needed, download a [packaged release](https://github.com/kframework/k/releases/)
     of the K Framework as part of their tool setup process.
 
+If you are interested in quickly trying out the K Framework without building
+from source, please see our
+[quick start guide](k-distribution/INSTALL.md#quick-start-guide-for-ubuntu-bionicfocal).
+
 The rest of this file assumes you intend to build and install the K Framework
 from source.
 
+## Contents
+
+1.  [Supported System Configurations](#supported-system-configurations)
+2.  [Prerequisite Install Guide](#prerequisite-install-guide)
+3.  [Build and Install Guide](#build-and-install-guide)
+4.  [IDE Setup](#ide-setup)
+5.  [Running the Test Suite](#running-the-test-suite)
+6.  [Changing the KORE Data Structures](#changing-the-kore-data-structures)
+7.  [Building the Final Release Directory/Archives](#building-the-final-release-directoryarchives)
+8.  [Compiling Definitions and Running Programs](#compiling-definitions-and-running-programs)
+9.  [Troubleshooting](#troubleshooting)
+
+# Supported System Configurations
+
+We release packages for the following system configurations given in order
+of kernel, CPU architecture, and operating system/package manager.
+These configurations are known to be able to build and run the K Framework:
+
+*   Linux (x86-64) Ubuntu Bionic Beaver
+*   Linux (x86-64) Ubuntu Focal Fossa
+*   Linux (x86-64) Arch Linux
+*   Linux (x86-64) Debian Buster
+*   Linux (x86-64) Platform Indepdent Binary (tested on Ubuntu)
+*   macOS (x86-64) Brew
+
+NOTE: users of the platform independent binary must install any needed
+runtime dependencies separately.
+
+The following system configurations have **no direct support** (but see notes
+below about virtual machine/emulation support):
+
+*   Windows (any architecture)
+*   macOS (M1)
+
+## Virtualized System Configurations
+
+Even if your system is not supported directly, you may be able to:
+
+1.  Install a free virtual machine/emulation software;
+2.  Install a supported Linux distribution on your virtual machine
+
+Our recommendations for virtual machine software are as follows:
+
+*   Windows 10 (x86-64) users should use
+    [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/)
+    with [version 2](https://docs.microsoft.com/en-us/windows/wsl/compare-versions) **recommended**
+    (version 1 should run the K Framework packages but has known issues when building from source).
+    To install a WSL Linux distribution, follow
+    [these instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+    Then begin the package installation/source build process in the newly created
+    Linux environment.
+
+*   Windows 8 and older (x86-64) users can install
+    [VirtualBox](https://www.virtualbox.org) and then:
+    1.   create a new virtual machine (VM);
```

Comment:
We do not need to include instructions in the K README about setting up virtual development environments. Users can google that on their own. You can include a single sentence or two saying that they will need to set it up, but instruct them to look up their own guides.